CONTRIBUTING.md

```diff
@@ -78,6 +78,35 @@ Some additional points:
+
+The benefits of being a core contributor include:
+- Increased influence of the direction of the project,
+- The ability to create branches in the main repository and merge your own code, and
+- Community recognition and visibility for their contributions and expertise.
+
+### Becoming a Core Contributor
+
+Core contributor candidates need to have a demonstrated proficiency with the Stratum codebase and a track record of code reviews.  Members of the Technical Steering Team (TST) and existing core contributors will regularly invite people to become new core contributors. Nominations can also be made (including self-nominations) to the TST at any time.
+
+A good nomination will include details about who the person is (including their Github username) and outline their experience with the Stratum codebase. Nominations are intended to start a conversation that results in a decision to make the person a core contributor – anyone whose nomination is not initially approved is encouraged to gain more experience with code submission and code review in order to gain further mastery over the codebase. Partial approval is also possible (e.g. a person may be granted the ability to triage issues, but not merge pull requests), and full approval may be granted after the contributor has a chance to gain more experience.
+
+New core contributors will be assigned a mentor that is either a TST member or existing core contributor. The mentor will serve as the primary point of contact to help onboard the new core contributors and answer any questions they have with their new responsibilities. The mentor is not the only point of contact, and core contributors should feel free to reach out to others if and when they have questions or concerns.
+
+### Tips for Core Contributors
+For your own contributions, you now have the ability to approve and merge your own code. For larger or potentially controversial pull requests, please give the community an opportunity (at least a few business days) to review your contribution. **With great power comes great responsibility; please don't abuse this privilege.**
+
+For branches in the main repository, please try to use sensible names and clean up these branches when they are no longer needed (e.g. after they are merged to main).
+
+Generally, Stratum follows [Google’s best practices for code review](https://google.github.io/eng-practices/review/reviewer/). Code reviews (particularly large ones) can be tricky to review, but it can be easiest to start with the big picture before diving into the details. Pull requests don’t have to be perfect before they are merged, but they shouldn’t introduce any regressions or break tests. It is important to get any issues with the high level design and interfaces resolved before merging code; issues in compartmentalized areas (e.g. implementation details) can sometimes be better addressed in follow-up contributions. 
```

Comment:
```suggestion
Generally, Stratum follows [Google’s best practices for code review](https://google.github.io/eng-practices/review/reviewer/). Code reviews (particularly large ones) can be tricky to review, but it can be easiest to start with the big picture before diving into the details. Pull requests don’t have to be perfect before they are merged, but they mustn't introduce any regressions or break tests. It is important to get any issues with the high level design and interfaces resolved before merging code; issues in compartmentalized areas (e.g. implementation details) can sometimes be better addressed in follow-up contributions. 
```

That whole section reads pretty apologetic of bad PRs. The eng guidelines are pretty clear on those things:

>100 lines is usually a reasonable size for a CL

> Note that **reviewers have discretion to reject your change outright for the sole reason of it being too large**.


> right size for a CL is **one self-contained change**

If, at review time, it's already evident that there is followup needed, why not fix it now?

Reply:
Thanks for the feedback, and I understand your point. I'll clean this up and point reviews to the best practices guide.